test/integration_test.py

```diff
@@ -713,3 +718,55 @@ def test_mini_dss_failures(self):
                 mini_dss.get_file(uuid, version, 'aws')
             with self.assertRaises(self.SpecialError):
                 mini_dss.get_native_file_url(uuid, version, 'aws')
+
+
+class AzulChaliceLocalIntegrationTest(AzulTestCase):
+    host = '127.0.0.1'
+    port = 8000
+    url = furl(scheme='http', host=host, port=port)
+    server = None
+    server_thread = None
+
+    @classmethod
+    def setUpClass(cls, config=config) -> None:
+        super().setUpClass()
+        app_module = load_app_module('service')
+        app_dir = os.path.dirname(app_module.__file__)
+        factory = chalice.cli.factory.CLIFactory(app_dir)
+        config = factory.create_config_obj()
+        cls.server = factory.create_local_server(app_obj=app_module.app,
+                                                 config=config,
+                                                 host=cls.host,
+                                                 port=cls.port)
+        cls.server_thread = threading.Thread(target=cls.server.serve_forever)
+        cls.server_thread.start()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        cls.server.shutdown()
+        cls.server_thread.join()
+        super().tearDownClass()
+
+    def test_local_chalice_health_endpoint(self):
+        url = self.url.copy()
+        url = url.set(path='health').url
```

Comment:
Reduce further here and where applicable.

Reply:
@hannes-ucsc I was unsure as to what was meant by reduce further. So I removed the `.url` bit and assigned it where the URL is actually used, is that valid?